### PR TITLE
Results in react

### PR DESCRIPTION
**Pull request by matzimowski**
it doesn't look pretty, maybe later..
screen shot 2016-02-14 at 22 28 02

### CODE REVIEW

**Comment by matzimowski** [comment]
@kdzwinel :koala:

File: js/components/results.js [hunk 1]
```diff
@@ -1,6 +1,65 @@
...
+            if(votes && votes[questionId]) {
+                for(var userId in votes[questionId]) {
+                    if(votes[questionId][userId] == correct) {
+                        results[userId] = results[userId] ? results[userId] + 1 : 1;
+                    }
+                }
+            }
+        }
+
+        var resultsArr = [];
```
Comment (kdzwinel) [inline js/components/results.js]: var

**Comment by matzimowski** [comment]
@kdzwinel up boy

File: js/components/results.js [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+        let questions = props.questions;
+        let votes = props.votes;
+        let usersArray = props.users;
+        let users = {};
+        let results = {};
+        let resultsArr = [];
+
+        usersArray.forEach((user) => users[user.id] = user);
+
+        for(var questionId in questions) {
```
Comment (kdzwinel) [inline js/components/results.js]: var var var

**Comment by kdzwinel** [comment]
:+1:  :fire:

**Pull request merged**